tsconfig.json

```diff
@@ -7,7 +7,8 @@
     "declaration": true,
     "esModuleInterop": true,
     "types": [
-      "node"
+      "node",
+      "mocha"
```

Comment:
It looks like it will load `@types/mocha` everywhere including `src`, I don't know if this is correct and acceptable